MOxUnit/util/moxunit_util_escape_xml.m

```diff
@@ -0,0 +1,33 @@
+function str_esc=moxunit_util_escape_xml(str)
```

Comment:
Escaping the XML special characters is definitely the correct way to go about this.

I was discarding the XML tags before because the error messages produced by MATLAB have a bunch of `<a>` links which point to where in the code the error is and I figured these would break (by not pointing anywhere) when they are in the XML test report. So it might be good to check whether this is the case and sanitize away just the `<a...>` and `<\a>` tags if necessary.